.github/workflows/poddefaults_docker_publish.yaml

```diff
@@ -13,30 +13,39 @@ on:
     paths:
       - components/admission-webhook/**
 
+env:
+  DOCKER_USER: kubeflownotebookswg
+  IMG: kubeflownotebookswg/admission-webhook
```

Comment:
@adilhusain-s could you change the image here to `kubeflownotebookswg/poddefaults-webhook`? This is the image we  were using